components/table/table-col-sort-button.js

```diff
@@ -84,17 +95,57 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 		super();
 		this.nosort = false;
 		this.desc = false;
+
+		this._describedById = getUniqueId();
 	}
 
 	static get focusElementSelector() {
 		return 'button';
 	}
 
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+
+		if (!this.dataType) {
+			console.warn('d2l-table-col-sort-button: data-type attribute does not have a value.');
+		}
```

Comment:
I added warnings for cases where there wasn't a `data-type`, but it's possible this would get too spammy, since nowhere will have it yet, and there also maybe could be cases where we would just want to use the fallback.